pkg/security/certs_test.go

```diff
@@ -138,6 +140,30 @@ func TestGenerateNodeCerts(t *testing.T) {
 	}
 }
 
+func generateAllCerts(certsDir string) error {
+	if err := security.CreateCAPair(
+		certsDir, filepath.Join(certsDir, security.EmbeddedCAKey), 512, time.Hour*48,
+	); err != nil {
+		return errors.Errorf("could not generate CA pair: %v", err)
```

Comment:
so the goal of this PR was to just add certificate rotation, #14703 is the big one.
I'm happy using this one instead and closing the other one, but the last commit here has the advantage of being self contained.